L/libgd/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+using BinaryBuilder, Pkg
+
+name = "libgd"
```

Comment:
```suggestion
name = "LibGD"
```
Note that you need to rename the directory to `LibGD`

Reply:
> Note that you need to rename the directory to `LibGD`

Reply:
Done!

Reply:
Still error.... with libiconv

Reply:
I'll have a look later

Reply:
@giordano This is the only unbuilt dependency of gnuplot 5.4.1 I guess!
After this, I'll try that, and try linking it to [Gnuplot.jl](https://github.com/gcalderone/Gnuplot.jl) and [Gaston.jl](https://github.com/mbaz/Gaston.jl)
Seriously, thanks a lot!!! 😀